src/main/java/Duke.java

```diff
@@ -1,10 +1,271 @@
+import java.util.Arrays;
+import java.util.Scanner;
+
 public class Duke {
+
+    // Constants
+    private static final String LOGO = " _     _                _           ____    ______   ______   ______ \n"
+            + "| |   | |      /\\      | |         / __ \\  / __   | / __   | / __   |\n"
+            + "| |__ | |     /  \\     | |        ( (__) )| | //| || | //| || | //| |\n"
+            + "|  __)| |    / /\\ \\    | |         \\__  / | |// | || |// | || |// | |\n"
+            + "| |   | | _ | |__| | _ | |_____      / /  |  /__| ||  /__| ||  /__| |\n"
+            + "|_|   |_|(_)|______|(_)|_______)    /_/    \\_____/  \\_____/  \\_____/\n";
+    private static final String LINE = "____________________________________________________________\n";
+    private static final String COMMAND_LIST = "list";
+    private static final String COMMAND_DONE = "done";
+    private static final String COMMAND_BYE = "bye";
+    private static final String COMMAND_TODO = "todo";
+    private static final String COMMAND_DEADLINE = "deadline";
+    private static final String COMMAND_EVENT = "event";
+    private static final String TIME_KEYWORD_BY = "/by";
+    private static final String TIME_KEYWORD_AT = "/at";
+
+    private static final TaskManager taskManager = new TaskManager();
+
+    /**
+     * Print sentences with line above and below the text block.
+     *
+     * @param sentences Sentence to be printed.
+     */
+    private static void blockPrint(String[] sentences) {
+        String printMessage = LINE + String.join("\n", sentences) + "\n" + LINE;
+        System.out.println(printMessage);
+    }
+
+    /**
+     * Print help message with valid commands.
+     */
+    private static void printHelp() {
```

Comment:
All in all, good naming of functions and variables. Functions are in the form of verbs. Good job!